modules/openshift-cluster-maximums-major-releases.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc
+
+[id="cluster-maximums-major-releases_{context}"]
+= {product-title} Tested cluster maximums for major releases
+
+Tested Cloud Platforms for OpemShift 3.X: OpenStack, Amazon Web Services and Azure.
```

Comment:
Tested cloud platforms for {product-title} 3.x: Red Hat OpenStack, Amazon Web Services, and Microsoft Azure.
Tested cloud platforms for {product-title}  4.x: Amazon Web Services, Microsoft Azure, and Google Cloud Platform.